opennft/opennft.py

```diff
@@ -101,6 +103,35 @@ def on_created(self, event):
             self.fq.put(event.src_path)
 
 
+class ViewBoxWithoutPadding(pg.ViewBox):
+    def suggestPadding(self, axis):
+        return 0.0
+
+
+class PluginWindow(QDialog):
```

Comment:
Maybe we could move this class to another module?

Reply:
Done in [commit 2a1353e](https://github.com/OpenNFT/OpenNFT/pull/63/commits/2a1353ecf38255b48d31e1fed6d1ee55996e8940)